### PR TITLE
Item charges: Add sacks of vegetables

### PR DESCRIPTION
**Pull request by tomcylke**
Adds sacks of vegetables to the item charges plugin

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java [hunk 1]
```diff
@@ -30,140 +30,15 @@
...
-import static net.runelite.api.ItemID.WATERING_CAN5;
-import static net.runelite.api.ItemID.WATERING_CAN6;
-import static net.runelite.api.ItemID.WATERING_CAN7;
-import static net.runelite.api.ItemID.WATERING_CAN8;
-import static net.runelite.api.ItemID.WATERSKIN0;
-import static net.runelite.api.ItemID.WATERSKIN1;
-import static net.runelite.api.ItemID.WATERSKIN2;
-import static net.runelite.api.ItemID.WATERSKIN3;
-import static net.runelite.api.ItemID.WATERSKIN4;
+import static net.runelite.api.ItemID.*;
```
Comment (zeruth) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java]: You should avoid wildcard imports. Make sure to configure your IDE to not do this in the future.

Comment (Ganom) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java]: > 
> 
> You should avoid wildcard imports. Make sure to configure your IDE to not do this in the future.

Wildcard is accepted for static imports iirc.

Comment (zeruth) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java]: It certainly isn't upstream. And everywhere else it isn't done like that. The project configuration by default won't allow it unless its 99+ imports.

Comment (Lucwousin) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java]: https://github.com/runelite-extended/runelite/wiki/Code-Conventions

```By changing Names count to use static imports with '*' to 10 (static imports for f.e. ItemIDs with wildcards are preferred)```

**Comment by tomcylke** [comment]
Even if it's not good practice it might be worth using wildcard imports in this instance because there are a shitton of them. The item charges plugin is also missing some things, so there will be more imports in the future.

**Pull request merged**